src/homepageExperience/containers/ArduinoWizard.tsx

```diff
@@ -177,13 +175,7 @@ export class ArduinoWizard extends PureComponent<{}, State> {
   render() {
     return (
       <Page>
-        <Page.Header fullWidth={false}>
-          {/* Need an empty div so the upgrade button aligns to the right. (Because clockface uses space-between to justifyContent)*/}
-          <div />
-          {!isFlagEnabled('multiOrg') && (
-            <RateLimitAlert location="firstMile.arduinoWizard" />
-          )}
-        </Page.Header>
+        <Page.Header fullWidth={false}></Page.Header>
```

Comment:
Although the page header no longer contains any elements, we don't want to remove it, because our page styles expect a header to exist. If we remove the header, the onboarding page looks like this. Way too close to the top.

https://user-images.githubusercontent.com/91283923/195183599-09e906ba-37a4-4de7-8775-4d25825be12e.mov

Instead, we want the page to keep looking like this.

https://user-images.githubusercontent.com/91283923/195183669-8e9cbd3f-4891-4c72-b66b-d0b32e87d7a5.mov

Reply:
minor thing but

```suggestion
        <Page.Header fullWidth={false} />
```

Reply:
Good idea. I'll do this for all the onboarding pages.